File: C++/Papier, nozyce, kamien.cpp
#include <string>
#include <iostream>

int main() 
{
    std::string gesture;
    std::cin >> gesture;
    if (gesture == "kamien")
        std::cout << "papier\n";
    else if (gesture == "nozyce")
        std::cout << "kamien\n";
    else if (gesture == "papier")
        std::cout << "nozyce";
    else
        std::cout << "przegrales\n";
    return 0;
}

File: Java/Papier, nozyce, kamien.java
import java.util.Scanner;

public class Main 
{
    public static void main(String[] args) 
    {
        Scanner input = new Scanner(System.in);
        String gesture = input.next().trim();
        switch (gesture) 
        {
            case "kamien":
                System.out.println("papier");
                break;
            case "nozyce":
                System.out.println("kamien");
                break;
            case "papier":
                System.out.println("nozyce");
                break;
            default:
                System.out.println("przegrales");
        }
        input.close();
    }
}